CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+9. :shipit: Rebase and merge (This will be done for you if you don't have contributor access)
+10. ✂️ Delete your branch (optional)
+
+Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant the project the rights to use your contribution. For details, visit https://cla.microsoft.com. When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. This needs to only be done once for any .NET Foundation OSS project.
+
+Some repositories, such as [Azure/azure-functions-host](https://github.com/Azure/azure-functions-host) include an automated integration test suite. This will only be run when a project maintainer has provided initial review and may involve your changes being moved to a test branch. It is encouraged that you set up an environment to test these yourself before submitting.
+
+If you have any questions about the process, please ask the project maintainers.
+
+### Labal glossary
+
+A common set of labels, described in this section, are used across repositories managed by the Azure Functions team. A given repository may use additional issues. Be sure to check that repository's contribution guide or ask a maintainer if you have any questions.
+
+| Label                | Description                                                                             |
+|----------------------|-----------------------------------------------------------------------------------------|
+| cla-already-signed   | The Contributor License Agreement has already been signed by the author                 |
+| cla-not-required     | The scope of the change does not require the signing of a Contributor License Agreement |
+| cla-required         | The author needs to sign a Contributor License agreement                                |
+| cla-signed           | The author has signed the Contributor License Agreement                                 |
+| community            | The author of the pull request is not one of the project maintainers                    |
+| do-not-merge         | The pull request should not be merged by a project maintainer                           |
+| pr-changes-requested | A project maintainer has requested changes and is awaiting updates from the author      |
+| pr-expired           | Requested changes were not provided within an expected time window.                     |
+| pr-in-review         | A project maintainer is reviewing the pull request                                      |
+
+## Creating documentation and samples
+
+There are a variety of ways you can contribute to Functions content. In addition to those discussed in this section, updates to repository readmes, wikis, or code comments are always welcome.
```

Comment:
Will outsiders have permission to update wiki content? I think they don't, unless there are some magic GitHub settings that I'm not aware of.

Reply:
Good point. Will remove.